serverlessworkflow/modules/ROOT/pages/cloud/common/_create_namespace_and_deploy_info.adoc

```diff
@@ -0,0 +1,45 @@
+
+--
+.Create namespace
+[source,shell]
+----
+kubectl create namespace serverless-workflow-greeting-quarkus
+----
+--
+
+. Set the Kubernetes context to the newly created namespace using the following command:
++
+--
+.Set Kubernetes context to a namespace
+[source,shell]
+----
+kubectl config set-context --current --namespace=serverless-workflow-greeting-quarkus
+----
+
+After setting the context, all kubectl commands will use the selected namespace. +
+You can use the following command to verify the current namespace:
+
+.Verify the current namespace
+[source,shell]
+----
+kubectl config view --minify -o jsonpath='{..namespace}'
+----
+--
+
+. Deploy your {context} application {deploy_application}.
++
+--
+The next step is to deploy your workflow application and execute it.
+You can read the further sections on the different procedures to deploy your {context} application.
+
+[NOTE]
+====
+You can use the native image due to the faster startup. +
+For more information about installing the workflow application, see xref:cloud/build-workflow-image-with-quarkus-cli.adoc[Building workflow images using Quarkus CLI] document.
+====
+--
+
+In the following procedures, you can find two examples of deploying your workflow application, including:
```

Comment:
what about Quarkus CLI ?
Reusable parts are good but not everything can be put in ...

Reply:
>> Reusable parts are good but not everything can be put in ...
What do you mean?

Reply:
well here it talks about
```
In the following procedures, you can find two examples of deploying your workflow application, including:
```
but the procedures are not referenced from the this file... so that is dangerous IMO

Reply:
actually, the procedures are not in this common file, but in the rendered file you can see that the next topics.

![Screenshot 2022-11-24 at 14 47 21](https://user-images.githubusercontent.com/1863987/203842918-598bc0bd-ce71-4f5c-be51-c32190c21b5f.png)


Need to add the quarkus-cli option as well.

Reply:
That is my problem. This text should be in the main file, not in an include. Else you have a tight-coupled between includes and this may lead to errors in the future ...

Reply:
Are you suggesting to move the `In the following...` paragraph to the main adoc file?

Reply:
I have updated it and added a anchor link to the sections in question.